cluster.go

```diff
@@ -174,6 +177,7 @@ func (c *Cluster) setupRPCClients() {
 	c.tracker.SetClient(c.rpcClient)
 	c.ipfs.SetClient(c.rpcClient)
 	c.api.SetClient(c.rpcClient)
+	c.proxy.SetClient(c.rpcClient)
```

Comment:
`for _, api := range apis { api.SetClients(c.rpcClient) }` and so on...

Reply:
We can use some tool that automates such code reviews such as golintci

Reply:
Those tools can also be run locally before pushing and you can automate that with things like [pre-commit](https://pre-commit.com/) as well. Plus in this particular case, it wouldn't have caught it as this comment is based on a requested change above.

Reply:
Will set it up locally and will look into pre-commit. thanks @lanzafame